projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -64,6 +76,13 @@ export class TabParagraphContainerComponent
     this.activeTabNum = tabNum;
   }
 
+  ngOnInit(): void {
```

Comment:
Is adding the "implementation" of `OnInit` a breaking change?

Reply:
I don't know :)

Reply:
I will ask